public/app/features/dashboard/components/DashboardSettings/SettingsCtrl.ts

```diff
@@ -182,6 +182,24 @@ export class SettingsCtrl {
     let confirmText = '';
     let text2 = this.dashboard.title;
 
+    if (this.dashboard.meta.provisioned) {
+      appEvents.emit('confirm-modal', {
+        title: 'Cannot delete provisioned dashboard',
+        text: `
+          Provisioned dashboards would be recreated again from the config file. Remove the dashboard from the
```

Comment:
> Provisioned dashboards would be recreated again from the config file. Remove the dashboard from the
config file to delete it.

How about
> This dashboard is managed by Grafanas provisioning and cannot be deleted. Remove the dashboard from the
config file to delete it.

I think its enough to say that they cannot delete it.